index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <section class="intro">
+
+<div class="logo">
+    <h1>Eros</h1>
+</div>
+  <div class="navbar">
+   <ul>
+       
+        <li><a href="Home">Home</a></li>
+        <li><a href="Porfolio">Portfolio</a></li>
+        <li><a href="Contact">Contact</a></li>
+
+    </ul>
+  </div>
+</section>
+<section >
+  <div class="description">
+    <img class="profile" src="./Images/Profile Photo.jpeg" alt="profile">
+    <h2 class="work"> I am Aime <br> a full stack software developer</h2>
+    <!-- <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur amet porro id, consectetur cumque enim earum illum officiis,</p> -->
+    <input class="btn" type="button" value="My Services">
+  </div>
+</section>
```

Comment:
- [ ] Please kindly use the `h1` for the main title (e.g., "I'm Marianne ...") and this will let google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.
- [ ] Please from the project requirements, you should stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A1288) design as much as possible. But as you can see you have some differences in the design. You can see the design below and compare it with your project design. :pray:

| Your Design |Figma Design|
| :---:   | :---: | 
|  ![Screenshot from 2022-10-31 11-34-36](https://user-images.githubusercontent.com/92303891/198998773-e7afdcfc-ea92-44ca-aab1-533c36ef52f6.png)| ![Screenshot from 2022-10-31 11-33-11](https://user-images.githubusercontent.com/92303891/198998599-ad5077f5-95dc-4772-9bad-7ec17a8fdd69.png) |